\doxysection{pip.\+\_\+internal.\+exceptions.\+Configuration\+Error Class Reference}
\hypertarget{classpip_1_1__internal_1_1exceptions_1_1_configuration_error}{}\label{classpip_1_1__internal_1_1exceptions_1_1_configuration_error}\index{pip.\_internal.exceptions.ConfigurationError@{pip.\_internal.exceptions.ConfigurationError}}
Inheritance diagram for pip.\+\_\+internal.\+exceptions.\+Configuration\+Error\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=4.000000cm]{classpip_1_1__internal_1_1exceptions_1_1_configuration_error}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}General exception in configuration\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
C\+:/\+Users/kmarty/\+Documents/\+Labo3\+\_\+architecture\+\_\+killian\+\_\+marty/\+Lib/site-\/packages/pip/\+\_\+internal/exceptions.\+py\end{DoxyCompactItemize}
